src/segments/sitecore.go

```diff
@@ -33,13 +33,16 @@ type UserConfig struct {
 }
 
 func (s *Sitecore) Enabled() bool {
-	if !s.env.HasFiles(sitecoreFileName) || !s.env.HasFiles(path.Join(sitecoreFolderName, userFileName)) {
+	if !s.env.HasFiles(sitecoreFileName) || !s.env.HasFilesInDir(sitecoreFolderName, userFileName) {
+		s.env.Debug("sitecore cli configuration files were not found")
 		return false
 	}
 
 	var userConfig, err = getUserConfig(s)
 
 	if err != nil {
+		s.env.Debug("unable to read 'user.json' file")
```

Comment:
this is absolete, the error will report where we are in the code as well.

Reply:
thanks, for the review

The log was removed.